dipy/align/imaffine.py

```diff
@@ -880,7 +891,8 @@ def __init__(self,
 
     def _init_optimizer(self, static, moving, transform, params0,
```

Comment:
docstring needs to be updated with information about new parameters, `static_mask` and `moving_mask`